### DISCUSSION TITLE
Show percent tasks complete in a new projects Beta field

### DISCUSSION THREAD

**Discussion by leighjboyd** [community/community#12050] [Projects and Issues]
In issues you can see number of tasks, and number complete.
Would be nice to see this from Projects Beta as well, without having to click down.

My suggestion:
 add a new field for this progress circle.

**Comment by Neudrino** [comment]
This issue is probably a **duplicate** of the most popular issue:
- :arrow_right: https://github.com/github-community/community/discussions/7575

Other (possible) duplicates are:
- https://github.com/github-community/community/discussions/16616
- https://github.com/github-community/community/discussions/13033
- https://github.com/github-community/community/discussions/16721
- https://github.com/github-community/community/discussions/12050
- https://github.com/github-community/community/discussions/15929

An **answer** from Github as of 2022-06-13 can be found at: https://github.com/github-community/community/discussions/16721#discussioncomment-2939031
